services/ui-src/src/components/overlays/EntityDetailsOverlayQualityMeasures.tsx

```diff
@@ -74,7 +71,7 @@ export const EntityDetailsOverlayQualityMeasures = ({
       </Heading>
       <Text sx={sx.reportSubtitle}>{overlayVerbiage.MCPAR.reportSubtitle}</Text>
       <Heading as="h3" sx={dashboardTitleStyling(canAddEntities)}>
-        {parseCustomHtml(overlayVerbiage.MCPAR.dashboardTitle)}
```

Comment:
I think we still need to parse the html here

<img width="750" height="465" alt="Image" src="https://github.com/user-attachments/assets/ecfe6fa0-d412-484d-9dad-af7d8b91397e" />